hooks/post_gen_project.py

```diff
@@ -116,11 +108,11 @@ def sort_blocsks(blocks):
     return package_blocks, command_blocks
     
     
-def insert_blocks(target_dir, package_blocks, command_blocks):
+def insert_blocks(target_dir, target_file, package_blocks, command_blocks):
     """
     Inserts LaTeX header blocks into the manuscript.tex file in the target directory after the \\documentclass line.
     """
-    target_path = Path(target_dir) / "manuscript.tex"
+    target_path = Path(target_dir) / target_file
```

Comment:
I think normally  we would just pass in the path to the file, not separate paths to the folder and then the filename.  Let's maybe just fix that quickly.